astropy/coordinates/tests/test_name_resolve.py

```diff
@@ -143,33 +143,25 @@ def test_names():
 
 @pytest.mark.remote_data
 def test_name_resolve_cache(tmpdir):
-    from astropy.utils.data import _get_download_cache_locs, get_cached_urls
-    import shelve
+    from astropy.utils.data import get_cached_urls
 
     target_name = "castor"
 
     temp_cache_dir = str(tmpdir.mkdir('cache'))
     with paths.set_temp_cache(temp_cache_dir, delete=True):
-        download_dir, urlmapfn = _get_download_cache_locs()
-
-        with shelve.open(urlmapfn) as url2hash:
-            assert len(url2hash) == 0
+        assert len(get_cached_urls()) == 0
 
         icrs1 = get_icrs_coordinates(target_name, cache=True)
 
-        # This is a weak test: we just check to see that a url is added to the
-        #  cache!
```

Comment:
No reason to remove this comment?

Reply:
It's not really clear in what sense the test is weak? It checks that after downloading the cache contains exactly the single URL that was requested and nothing else.

Reply:
No idea. I think @adrn wrote that comment.

Reply:
Alright, let's just remove it as you did. We can always un-weak it in a follow-up PR if needed. Saying the test is weak isn't doing much anyway.